src/rqt_gauges_2/steering_wheel_gauge.py

```diff
@@ -101,6 +100,21 @@ def updateValue(self, value: float):
         self.value = value
         self.repaint()
 
+    def setMaxValue(self, max_value):
+        # Modifies the maximum value of the gauge
+        # Args:
+        #   max: Value to update the maximum value of the gauge.
+        if self.value > max_value:
+            self.value = max_value
+        if max_value <= self.minValue:
```

Comment:
Just checking, `min_value` and `max_value` cannot be the same?

Reply:
yes, then we can create a symmetric steering wheel